src/disco/components/Addon/index.js

```diff
@@ -137,30 +139,30 @@ export class AddonBase extends React.Component {
 
       let imageUrl = getPreviewImage(addon);
 
-      if (!imageUrl && type === ADDON_TYPE_THEME) {
+      let headerLinkProps = {
+        className: 'theme-image',
+        href: '#',
```

Comment:
we should probably link to the static theme, we have it no?

Reply:
the link here is supposed to install/enable (like lightweight theme currently does)

Reply:
even if we disable the mozAddonManager?

Reply:
well right now..the way it's set up it wouldn't do anything. I guess we could link the header in this case to the addon detail page

On a side but related note if mozAddonManager is false, then this value would be passed to the buttons and I don't see the buttons working here either in this case. Is this expected?

Reply:
The install button works with or without the `mozAddonManager` by offering the link to the extension in a `href` prop.

Reply:
I think what we could do is that if `hasAddonManager` is `false`, then we do not surround the `<img />` with a link. No link, no action, for both lightweight and static themes.

That would also allow you to get rid of the test in `installStaticTheme()`.